TS2307 error while type checking

Hey there! Thank you for the great work on this lib. We've been implementing it in an app to simplify some of the font handling code and noticed that type checking started failing:

```
husky > pre-commit (node v14.15.0)
node_modules/react-native-font-faces/dist/typescript/utilities/enableFontFaces.d.ts:2:39 - error TS2307: Cannot find module 'src/types/EnableFontFacesResult' or its corresponding type declarations.
2 import { EnableFontFacesResult } from 'src/types/EnableFontFacesResult';
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/react-native-font-faces/dist/typescript/utilities/getDynamicFontList.d.ts:2:33 - error TS2307: Cannot find module 'src/types/DynamicFontList' or its corresponding type declarations.
2 import { DynamicFontList } from 'src/types/DynamicFontList';
                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/react-native-font-faces/dist/typescript/utilities/getExpoFontMap.d.ts:2:29 - error TS2307: Cannot find module 'src/types/ExpoFontMap' or its corresponding type declarations.
2 import { ExpoFontMap } from 'src/types/ExpoFontMap';
                              ~~~~~~~~~~~~~~~~~~~~~~~
```

This usually happens due to the usage of non relative paths, see:

https://cdn.jsdelivr.net/npm/react-native-font-faces@4.1.0/dist/typescript/utilities/getDynamicFontList.d.ts

--- a/package.json
+++ b/package.json
@@ -10,6 +10,7 @@
   "repository": "https://github.com/Kjens93/react-native-font-faces.git",
   "description": "Easily emulate @font-face behavior in react-native",
   "files": [
+    "src",
     "dist"
   ],
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -10,13 +10,13 @@
   "repository": "https://github.com/Kjens93/react-native-font-faces.git",
   "description": "Easily emulate @font-face behavior in react-native",
   "files": [
-    "src",
     "dist"
   ],
   "scripts": {
     "test": "jest",
     "build": "bob build",
-    "release": "semantic-release"
+    "release": "semantic-release",
+    "prepack": "yarn build"
   },
   "dependencies": {
     "redent": "^3.0.0",

--- a/src/utilities/enableFontFaces.tsx
+++ b/src/utilities/enableFontFaces.tsx
@@ -4,7 +4,7 @@ import { overrideTextRenderFn } from './overrideTextRenderFn';
 import { registerGlobalFontFaces } from './registerGlobalFontFaces';
 import { overrideTextInputRenderFn } from './overrideTextInputRenderFn';
 import { getDynamicFontList } from './getDynamicFontList';
-import { EnableFontFacesResult } from 'src/types/EnableFontFacesResult';
+import { EnableFontFacesResult } from '../types/EnableFontFacesResult';
 
 export function enableFontFaces(fontFaces: FontFace[]): EnableFontFacesResult {
   overrideTextRenderFn();

--- a/src/utilities/enableFontFaces.web.tsx
+++ b/src/utilities/enableFontFaces.web.tsx
@@ -1,5 +1,5 @@
 import { FontFace } from '../types/FontFace';
-import { EnableFontFacesResult } from 'src/types/EnableFontFacesResult';
+import { EnableFontFacesResult } from '../types/EnableFontFacesResult';
 import { generateInlineStyleSheet } from './generateInlineStyleSheet';
 
 export function enableFontFaces(fontFaces: FontFace[]): EnableFontFacesResult {

--- a/src/utilities/getDynamicFontList.ts
+++ b/src/utilities/getDynamicFontList.ts
@@ -1,5 +1,5 @@
 import { FontFace } from '../types/FontFace';
-import { DynamicFontList } from 'src/types/DynamicFontList';
+import { DynamicFontList } from '../types/DynamicFontList';
 import { getDynamicFontListEntry } from './getDynamicFontListEntry';
 
 export function getDynamicFontList(fontFaces: FontFace[]): DynamicFontList {

--- a/src/utilities/getDynamicFontListEntry.ts
+++ b/src/utilities/getDynamicFontListEntry.ts
@@ -1,7 +1,7 @@
 import { FontFace } from '../types/FontFace';
 import { explodeToArray } from './explodeToArray';
 import { getLocalFontName } from './getLocalFontName';
-import { DynamicFontListEntry } from 'src/types/DynamicFontListEntry';
+import { DynamicFontListEntry } from '../types/DynamicFontListEntry';
 
 export function getDynamicFontListEntry(fontFace: FontFace): DynamicFontListEntry | undefined {
   for (const source of explodeToArray(fontFace.src)) {

--- a/src/utilities/getExpoFontMap.ts
+++ b/src/utilities/getExpoFontMap.ts
@@ -1,5 +1,5 @@
 import { FontFace } from '../types/FontFace';
-import { ExpoFontMap } from 'src/types/ExpoFontMap';
+import { ExpoFontMap } from '../types/ExpoFontMap';
 import { getLocalFontName } from './getLocalFontName';
 import { getExpoFontMapEntry } from './getExpoFontMapEntry';
 

--- a/src/utilities/registerGlobalFontFaces.tsx
+++ b/src/utilities/registerGlobalFontFaces.tsx
@@ -1,4 +1,4 @@
-import { FontFace } from 'src/types/FontFace';
+import { FontFace } from '../types/FontFace';
 import { globalFontFaces } from './globalFontFaces';
 import { getLocalFontName } from './getLocalFontName';
 

